File: Python/1064.py
num_pos = 0
pos = []
media = 0
for i in range(6):
    num = float(input())
    if num > 0:
        num_pos += 1
        pos.append(num)

for num in pos:
    media += num
print(f'{num_pos} valores positivos')
print(f'{media/num_pos:.1f}')

File: C/1064.c
#include <stdio.h>
 
int main() {
    int quant = 0;
    float total = 0.0;
    float media = 0.0;
    for(int i = 1; i <= 6; i++){
        float val;
        scanf("%f", &val);
        if(val >= 0){
            total = total + val;
            quant ++;
        }
    }
    media = total / quant;
    printf("%d valores positivos\n", quant);
    printf("%.1f\n", media);
}
